terraform/environment/opensearch.tf

```
@@ -0,0 +1,79 @@
+data "aws_caller_identity" "current" {
+  provider = aws.eu_west_1
+}
+
+data "aws_vpc_endpoint_service" "opensearch" {
```

sorry, this should be
```suggestion
data "aws_vpc_endpoint" "opensearch" {
```